### ISSUE TITLE
Readonly Fields looks the same as non readonly on Content Publishing block.

### ISSUE THREAD

**Issue by joomlacorner** [joomla-projects/j4adminui#141]
### Steps to reproduce the issue
Content -> Add New Article 
Click on Publishing Block

### Expected result

It should display a disabled-ish state for the input since it is readonly.

### Actual result
![Screen Shot 2562-12-03 at 22 42 16](https://user-images.githubusercontent.com/144291/70066563-7e7add80-161f-11ea-835a-a2f3b46a8a01.png)

**Comment by kawshar** [comment]
Already there is a fix https://github.com/joomla-projects/j4adminui/pull/89

**Comment by joomlacorner** [comment]
@kawshar No. after apply that fix. it's not changed on this block.  Can you recheck?

**Comment by kawshar** [comment]
@sahidhossen please have a look.

**Comment by sahidhossen** [comment]
@joomlacorner  Please check the #89 commit. I have change the disabled and read-only field background colour. Hope this colour will satisfied with disabled-ish state.
